dracut/30ignition/ignition-fetch.service

```diff
@@ -0,0 +1,20 @@
+[Unit]
+Description=Ignition (fetch)
+DefaultDependencies=false
+Before=ignition-complete.target
```

Comment:
We don't have this in `spec2x` yet....trying a patch but for now I'd make this match the other units.